src/main/java/edu/greenblitz/robotName/commands/swerve/RotateByScoringMode.java

```diff
@@ -0,0 +1,17 @@
+package edu.greenblitz.robotName.commands.swerve;
+
+import edu.greenblitz.robotName.ScoringModeSelector;
+import edu.greenblitz.robotName.commands.swerve.rotateTo.RotateToAmp;
+import edu.greenblitz.robotName.subsystems.swerve.chassis.ChassisConstants;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+
+public class RotateByScoringMode extends ConditionalCommand {
+
+    public RotateByScoringMode() {
+        super(
+                new MoveByJoystickAndRotateToSpeaker(ChassisConstants.DRIVE_MODE),
+                new RotateToAmp(),
```

Comment:
We should also have a MoveByJoystickAndRotateToAmp
and maybe even MoveByJoystickAndRotateToTrap if we figure out how to tell it which one

Reply:
there is now for amp, climb will be in another pr